RELEASE-CHECKLIST.md

```diff
@@ -18,6 +18,8 @@ Release checklist:
   ```bash
   gem tag && gem release
   ```
+
+  (alternatively if gem-release isn't installed you can use `rake release`)
```

Comment:
Or just always use `bundle exec rake release`?
`rake release` tags as well.

Reply:
Ok agreed - in that case can drop reference to installing gem-release gem, which simplifies the readme (latest commit)